README.md

```diff
@@ -187,3 +188,20 @@ the car manually using the controller, use the following controls:
 
 The car starts in START mode. When pressed it will switch to MANUAL mode. Every
 subsequent press toggles between AUTO and MANUAL mode.
+
+
+Testing
+-------
+
+Testing is done with pytest. To run tests, make sure that the environment is
+initialized (run `rosvenv`, `rosinit`, and `rosdevel`) then run `pytest
+buzzmobile/tests`. This will run all tests in the tests directory. Be careful,
+beacuse there is a virtualenv in the root directory of the project, **running
+`pytest` will attempt to run thousands of unittests included with the python
+interpreter.**
+
+To write additional tests, please place within `buzzmobile/tests` in directories
+that match the source files, that is tests for the
+`buzzmobile/process/gps_mapper` node should go in
+`buzzmobile/tests/process/test_gps_mapper.py`. Integration and simulation tests
+should go in the `<TODO>` and `<TODO>` directories respectively.
```

Comment:
What do you think about `buzzmobile/tests/integration/test_name.py`? Or maybe `buzzmobile/integration_tests/test_name.py`. Similar for simulation